taichi/runtime/llvm/runtime.cpp

```diff
@@ -1036,6 +1036,18 @@ f32 cuda_shfl_up_sync_f32(u32 mask, f32 val, i32 delta, int width) {
   return 0;
 }
 
+bool cuda_any_sync(u32 mask, bool bit) {
+  return false;
+}
+
+int32 cuda_any_i32(int32 predicate) {
+  return (int32)cuda_any_sync(UINT32_MAX, (bool)predicate);
```

Comment:
I am not sure if these functions should return 0 or something else, we need @strongoier  and @qiao-bo to have a look.

Reply:
This is fine here. Since only `cuda_any_sync()` is patched later in llvm_context.cpp